migration-engine/connectors/sql-migration-connector/src/sql_migration.rs

```diff
@@ -110,6 +109,29 @@ pub enum TableChange {
     },
 }
 
+impl TableChange {
```

Comment:
When I see this, I always get a tiny headache. One of the problems with enums, and there's not much we can do without a complete refactoring.

Reply:
At least here you just take a reference, but sometimes you must take the ownership in these methods and, well, sigh...

Reply:
Agreed — I'm trying to refactor things to be flatter in general in the migration engine. This module can't be touched too much because the current migrate relies on it.